management-common-v2/src/main/java/org/terracotta/management/resource/AgentEntityV2.java

```diff
@@ -1,6 +1,7 @@
 /*
- * Copyright Terracotta, Inc.
- * Copyright Super iPaaS Integration LLC, an IBM Company 2024
+ * All content copyright (c) 2003-2012 Terracotta, Inc., except as may otherwise be noted in a separate copyright
```

Comment:
Why the different notice here and not just leave the original `Copyright Terracotta, Inc.`?

Reply:
In the prev PR for copyright, I have removed this year line with - `Copyright Terracotta, Inc.`
In review comment has been asked to preserve the year line(fixed for mentioned place, but forgot to do it at all the files)
So reverting last time removal of this year line , here is the previous [PR(PR-25)](https://github.com/Terracotta-OSS/management-common/pull/24) for your refrence